server/lib/fetch.py

```diff
@@ -211,7 +215,7 @@ def point_within_core(ancestor_entity,
 
 
 def series_core(entities, variables, all_facets, facet_ids=None):
```

Comment:
It seems this changes the returned structure, so to confirm, are all callers of series_core good?

Reply:
yeup, all callers of series_core are good